targets/csharp/source/PlayFabSDK/WindowsFormsApp1/Form1.cs

```diff
@@ -0,0 +1,115 @@
+using PlayFab;
+using PlayFab.ClientModels;
+using PlayFab.UUnit;
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace WindowsFormsApp1
+{
+    public partial class Form1 : Form
+    {
+        public Form1()
+        {
+            InitializeComponent();
+        }
+
+        public class CsSaveRequest
+        {
+            public string customId;
+            public TestSuiteReport[] testReport;
+        }
+
+        public int Main2(string[] args)
+        {
+            int result = 1;
+            bool taskFinished = false;
+            try
+            {
+                var mainTask = MainTask(args);
+                taskFinished = mainTask.Wait(120000);
```

Comment:
Is it possible to make the previously mentioned named variable shared to be used in here? Or would that not make sense?

Reply:
They're actually separate projects, so I can't link them together. But I can make it an independent const.